### PR TITLE
Bibtex updates

### PR DESCRIPTION
**Pull request by danielru**
A number of fixes and additions to the pint.bib file.

[Review on Reviewable](https://reviewable.io/reviews/parallel-in-time/parallel-in-time.github.io/48)

### CODE REVIEW

File: _bibliography/pint.bib [hunk 1]
```diff
@@ -1904,6 +1948,19 @@ @article{MinionEtAl2015
 	year = {2015}
 }
 
+@inproceedings{RuprechtEtAl2015_DDM,
+	author="Ruprecht, D. and Speck, R. and Krause, R.",
```
Comment (torbjoernk) [inline _bibliography/pint.bib]: use `{` instead of `"`

File: _bibliography/pint.bib [hunk 2]
```diff
@@ -1881,6 +1874,23 @@ @misc{Ariel2015
...
 
 @article{ArteagaEtAl2015,
+	abstract = { In view of the rapid rise of the number of cores in modern
+supercomputers, time-parallel methods that introduce concurrency along
+the temporal axis are becoming increasingly popular.  For the solution
+of time-dependent partial differential equations, these methods can
+add another direction for concurrency on top of spatial
+parallelization.  The paper presents an implementation of the
+time-parallel Parareal method in a C++ domain specific language for
+stencil computations (STELLA).  STELLA provides both an OpenMP and a
```
Comment (torbjoernk) [inline _bibliography/pint.bib]: you might need to enclose the abstract in double `{{` to make 'STELLA' appearing in capital letters after BibTeX processing (not applicable on website).

File: _bibliography/pint.bib [hunk 3]
```diff
@@ -1926,7 +1983,24 @@ @article{Song2015
 	year = {2015}
 }
 
+@inproceedings{SpeckEtAl2015_DDM,
+	author="Speck, R. and Ruprecht, D. and Minion, M. and Emmett, M. and Krause, R.",
```
Comment (torbjoernk) [inline _bibliography/pint.bib]: `{` vs. `"`

File: _bibliography/pint.bib (hunk 1 above)
Comment (danielru) [inline _bibliography/pint.bib]: Fixed.

File: _bibliography/pint.bib (hunk 2 above)
Comment (danielru) [inline _bibliography/pint.bib]: Fixed.

File: _bibliography/pint.bib (hunk 3 above)
Comment (danielru) [inline _bibliography/pint.bib]: Fixed. Also added authors' surnames.

**Pull request merged**